### PR TITLE
Improve design ca1 to improve basics aa3

### PR DESCRIPTION
**Pull request by schettn**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Documentation (non-breaking change which adds documentation)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently the design seems a bit outdated and there are several minor issues. #90 

**What is the new behavior (if this is a feature change)?**
- Now the design is more rounded and improved in general. Project styling, navbar styling,.. has been updated. #90

**Other information**:
- 🐍

### CODE REVIEW

**Review by Theralun — APPROVED** [review]
Beaut

File: src/components/atoms/charts/LanguageBar/index.jsx [hunk 1]
```diff
@@ -17,41 +17,53 @@ class LanguageBar extends React.Component {
   renderBars = (languages) => {
     let latest = 0;
 
-    return languages.map((language, i) => {
-      const { color, share } = language;
-      const value = latest + share;
+    console.log("LANG", languages);
```
Comment (Aichnerc) [inline src/components/atoms/charts/LanguageBar/index.jsx]: Is this necessary?
```suggestion

```

**Review by Aichnerc — APPROVED** [review]
Found a console.log which could be problematic.

File: src/components/atoms/charts/LanguageBar/index.jsx (hunk 1 above)
Comment (schettn) [reply src/components/atoms/charts/LanguageBar/index.jsx]: Nep

File: src/components/atoms/charts/LanguageBar/index.jsx [hunk 2]
```diff
@@ -17,7 +17,6 @@ class LanguageBar extends React.Component {
   renderBars = (languages) => {
     let latest = 0;
```
Comment (schettn) [inline src/components/atoms/charts/LanguageBar/index.jsx:19]: Remove this newline
```suggestion
```

**Pull request merged**